pygmo/core.cpp

```diff
@@ -357,6 +358,37 @@ PYBIND11_MODULE(core, m)
         },
         pygmo::batch_random_decision_vector_docstring().c_str(), py::arg("prob"), py::arg("n"));
 
+    // Genetic operators
```

Comment:
You have to make these expositions conditional on the pagmo version in use, otherwise half of the builds on the CI will keep on failing.

Reply:
I did it via the preprocessor directive ... but like this these lines will not be tested!